### ISSUE TITLE
HF0.2, wallet, list_my_accounts fails

### ISSUE THREAD

**Issue by gropox** [GolosChain/golos#263]
**list_my_accounts** shows an error

```
10 basic_exception: tion
this->symbol == o.symbol: 
    {}
    th_a  asset.hpp:195 operator+=
```

Balances are not shown

Problably default constructor is not properly initializes asset

https://github.com/GolosChain/golos/blob/golos-v0.17.0/libraries/wallet/wallet.cpp#L800

**Comment by gropox** [comment]
help does is not wrking also

**Comment by Nemo1369** [comment]
@gropox Issue was introduced by platform-dependent asset symbol initialisation fixes. It is known and in process now.

**Comment by Nemo1369** [comment]
Fixed with #232
